index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+
+    <header>
+        <nav id="phone">
+            <div class="logo">My Logo</div>
+            <img src="images/humbarger.svg" alt="humbarger" class="humbarger">
+        </nav>
+    </header>
+    <main>
+
+        <section id="bg">
+            <section id="bio">
+                <h1 class="title">I’m Mariane Glad to see you!</h1>
+                <article class="about">I’m a software developer! I can help you build a product , feature or website
```

Comment:
As per the requirement, `I'm a software... ` should be wrapped inside `<p></p>` tag. 
- [ ] Can you please change `<article>` tag  to `<p>` ? 👍🏽 
- [ ] Kindly test the UI and please fix the styles if its broken. 🟢 💯